classes/Gems/MenuNew/RouteLinkItem.php

```diff
@@ -54,10 +54,18 @@ public function renderContent(): string
 
         $url = $menu->routeHelper->getRouteUrl($this->name, $this->openParams);
 
```

Comment:
Waarvoor is dit nodig, hebben route-link-items children die in de twig gebruikt worden?